week-1/errors/4.js

```diff
@@ -1,2 +1,10 @@
-const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+// const 12HourClockTime = "20:53";
+// const 24hourClockTime = "08:53";
+
+// The above variables disobey acceptable naming convention by starting with numbers
+
+const twelveHourClockTime = "20:53";
+const twentyFourHourClockTime = "08:53";
```

Comment:
good, but I think you've got them the wrong way round - 20:53 is a 24 hour clock time